packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/index.tsx

```diff
@@ -86,12 +85,38 @@ const WorkspaceStatus = ({
 }) => {
   const isOnline = useSystemOnline();
 
-  // todo: finish display sync status
-  const [forceSyncStatus, startForceSync] = useDatasourceSync(
-    currentWorkspace.blockSuiteWorkspace
-  );
-
-  const setIsHovered = useSetAtom(hoverAtom);
+  const [syncEngineStatus, setSyncEngineStatus] = useState<
+    'syncing' | 'synced' | 'error'
+  >('syncing');
+
+  // FIXME: This is a hack to get the sync engine, we need refactor this in the future.
```

Comment:
does it make sense that we make sync engine a workspace space level state, along side with `useCurrentWorkspace()`?